src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj

```diff
@@ -22,6 +22,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <IncludeSymbols>true</IncludeSymbols>
     <Platforms>x64</Platforms>
+    <Version>2.4.1</Version>
```

Comment:
I changed the version in the Properties of the project. I'm thinking I should see a red line and a green line? Maybe I did the version change wrong.

Reply:
You're right about the event, that's a good replacement. Whenever possible, I like to keep backward-compatible for existing users.

I use <VersionPrefix>2.4.0</VersionPrefix>
Which has the same effect.

Please call the original pr version 3.0.0 to indicate a major breaking change in behavior.  
Thank you for your efforts!